docs/overrides.rst

```diff
@@ -0,0 +1,78 @@
+.. _`overrides`:
+
+=========
+Overrides
+=========
+This functionality opens the door to configure specific Page Objects depending
+on the request URL domain. Please have a look to :ref:`intro-tutorial` to
+learn the basics about overrides before digging deeper in the content of this
+page.
+
+Page Objects refinement
+=======================
+
+Any ``Injectable`` or page input can be overridden. But the overriding
+mechanism stops for the children of any already overridden type. This opens
+the door to refining existing Page Objects without getting trapped in a cyclic
+dependency. For example, you might have an existing Page Object for book extraction:
+
+.. code-block:: python
+
+    class BookPage(ItemWebPage):
+        def to_item(self):
+            ...
+
+Imagine this Page Object is provided by an external library, so you cannot
+directly modify it. But you want to extract an additional attribute (e.g. ``ISBN``) that
+was not extracted by the original Page Object. No problem, you can just override it
+using the following Page Object:
```

Comment:
This particular example can be solved in a more "standard" way with inheritance: inherit from BookPage, call super() in to_item. What do you think about adding a note about this, or providing a different example (not sure which though)?

Reply:
Good point. I've changed the example. Now you cannot use inheritance, because the original PO doesn't have any HTML at all. Changed here:

https://github.com/scrapinghub/scrapy-poet/pull/43/commits/b1bb4c406fcd4af077f0a82c39bc81b1d3478e40

Reply:
Great point! I've changed the example, and now inheritance is not enough. The change is here: https://github.com/scrapinghub/scrapy-poet/pull/43/commits/b1bb4c406fcd4af077f0a82c39bc81b1d3478e40